doc/source/ray-overview/installation.rst

```diff
@@ -96,19 +96,21 @@ You can install the nightly Ray wheels via the following links. These daily rele
         =============================================== ================================================
                Linux (x86_64)                                   Linux (arm64/aarch64)
         =============================================== ================================================
+        `Linux Python 3.11 (x86_64)`_                    `Linux Python 3.11 (aarch64)`_
         `Linux Python 3.10 (x86_64)`_                    `Linux Python 3.10 (aarch64)`_
         `Linux Python 3.9 (x86_64)`_                     `Linux Python 3.9 (aarch64)`_
-        `Linux Python 3.11 (x86_64)`_                    `Linux Python 3.11 (aarch64)`_
+        `Linux Python 3.12 (x86_64) (EXPERIMENTAL)`_     `Linux Python 3.12 (aarch64) (EXPERIMENTAL)`_
```

Comment:
the order feels a bit weird honestly. I kind of understand, but probably would still prefer it showing in the order of 39 310 311 312

or maybe add another `=========`? I do not know..